functions/src/index.ts

```diff
@@ -99,3 +99,101 @@ export const scheduledDeleteOfDocumentsSetToBeDeleted = region('us-central1')
         }
         return null
     })
+
+export const getOwnersDataByUIDs = https.onCall(async (data, context) => {
+    if (!context.auth) {
+        throw new https.HttpsError(
+            'failed-precondition',
+            'The function must be called while authenticated.',
+        )
+    }
+
+    const ownerUIDs = data.ownersList.map((id: string) => ({ uid: id }))
+
+    try {
+        const usersResult = await auth().getUsers(ownerUIDs)
+        const ownersData = usersResult.users.map((user) => ({
+            uid: user.uid,
+            email: user.email,
+        }))
+        return ownersData
+    } catch {
+        throw new https.HttpsError(
+            'invalid-argument',
+            'Could not get requested email addresses.',
+        )
+    }
+})
+
+export const getOwnerDataByEmail = https.onCall(async (data, context) => {
```

Comment:
Her bør me unngå at brukarar kan scanne e-postadresser for å finne ut kven som har konto på Tavla. F.eks. går det an å sjekke om den etterspurte e-postadressa (`data.email`) er i `owners` eller `ownerRequestRecipients` på minst éi tavla som funksjons-kallaren eig.